projects/components/src/splitter/splitter.component.scss

```diff
@@ -0,0 +1,81 @@
+@import 'mixins';
+
+:host {
+  width: 100%;
+  height: 100%;
+}
+
+.splitter-container {
+  display: flex;
+  width: 100%;
+  height: 100%;
+
+  &.horizontal {
+    flex-direction: row;
+  }
+
+  &.vertical {
+    flex-direction: column;
+  }
+
+  .splitter-content {
+    display: flex;
+  }
+
+  .splitter-content > p {
+    width: 100%;
+    height: 100%;
+  }
+
+  .splitter {
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
nit: can be replaced with `center-contents` mixin

Reply:
I prefer to type it out for better readability.